tools/d2g/d2g.go

```diff
@@ -357,10 +357,10 @@ func runCommand(containerName string, dwPayload *dockerworker.DockerWorkerPayloa
 		command.WriteString(" --security-opt=seccomp=unconfined")
 	}
 	command.WriteString(createVolumeMountsString(dwPayload, wdcs))
+	command.WriteString(envMappings(dwPayload))
 	if dwPayload.Features.TaskclusterProxy {
-		command.WriteString(" --add-host=taskcluster:127.0.0.1 --net=host")
+		command.WriteString(" --add-host=taskcluster:host-gateway -e TASKCLUSTER_PROXY_URL=${TASKCLUSTER_PROXY_URL/localhost/taskcluster}")
```

Comment:
I'm not familiar with adding this `host-gateway`. Do you have any tasks you made manually that uses this that the container has access to the tc proxy?

Reply:
That would need a g-w with the first patch here, for the proxy to listen on the docker interface.

Per https://docs.docker.com/reference/cli/docker/container/run/#add-host "The `--add-host` flag supports a special `host-gateway` value that resolves to the internal IP address of the host. This is useful when you want containers to connect to services running on the host machine."

Reply:
Thanks 👍🏻 Are you planning on doing the generic worker/tc proxy patch too? We can assist however you'd like/need.